tests/integration/use_cases/test_project.py

```diff
@@ -0,0 +1,18 @@
+from kili.adapters.kili_api_gateway import KiliAPIGateway
+from kili.use_cases.project.project import ProjectUseCases
+
+
+def test_when_create_project_it_works(kili_api_gateway: KiliAPIGateway):
+    kili_api_gateway.create_project.return_value = "fake_project_id"
+
+    # When
+    project_id = ProjectUseCases(kili_api_gateway).create_project(
+        input_type="TEXT",
+        json_interface={},
+        title="test",
+        description="description",
+        project_type=None,
+    )
+
+    # Then
+    assert project_id == "fake_project_id"
```

Comment:
how does it play with that? since it is mocked , the gateway it should not be able to fetch the project then return
```for attempt in Retrying(
            stop=stop_after_delay(60),
            wait=wait_fixed(1),
            retry=retry_if_exception_type(NotFound),
            reraise=True,
        ):
            with attempt:
                _ = self._kili_api_gateway.get_project(project_id=project_id, fields=["id"])
                ```

Reply:
```python
_ = self._kili_api_gateway.get_project(project_id=project_id, fields=["id"])
```

will return a magic mock, so it won't raise anything, so tenacity won't retry